cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/token/RangeUtils.java

```diff
@@ -30,13 +30,15 @@
 import com.google.common.collect.Multimap;
 import com.google.common.collect.Range;
 
-import org.apache.cassandra.spark.common.model.CassandraInstance;
+import org.apache.cassandra.bridge.TokenRange;
+import org.apache.cassandra.spark.data.model.TokenOwner;
 import org.apache.cassandra.spark.data.partitioner.Partitioner;
+import org.jetbrains.annotations.NotNull;
 
 /**
  * Common Cassandra range operations on Guava ranges. Assumes ranges are not wrapped around.
- * It's the responsibility of caller to unwrap ranges. For example, [100, 1] should become
- * [100, MAX] and [MIN, 1]. MIN and MAX depend on {@link Partitioner}.
+ * It's the responsibility of caller to unwrap ranges. For example, {@code (100, 1]} should become
+ * {@code (100, MAX]} and {@code (MIN, 1]}. MIN and MAX values depend on {@link Partitioner}.
```

Comment:
correct the notion that the range is open on the lower end and closed on the upper end.